src/ios/CDVInAppBrowser.m

```diff
@@ -49,86 +51,122 @@ - (void)open:(CDVInvokedUrlCommand*)command
         return;
     }
     self.usewkwebview = browserOptions.usewkwebview;
-    if(self.usewkwebview){
-        [[CDVWKInAppBrowser getInstance] open:command];
-    }else{
-        [[CDVUIInAppBrowser getInstance] open:command];
-    }
+    #if WK_WEB_VIEW_ONLY
```

Comment:
I personally find the number of these conditional blocks to be quite a bit ugly. I suspect there should be some clean way to abstract out `[CDVWKInAppBrowser getInstance]` vs `[CDVUIInAppBrowser getInstance]`.

Reply:
I am not a fan of this build flag, too. It is kind of ugly and hard to get while reading this code (for non-objective-c devs) but I think it is great for an interim solution until #535 gets released in a **major** release hopefully soon.